### ISSUE TITLE
AMANDA Accounts: CPO Office (NN)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#3311]
**From Greg P:**

Here there y’all,
I am needing to get access to AMANDA for a couple of people within CPO. CPO is sponsored by ATD, so I figured I would reach out and ask where do I start these days?
1.	Randahl.Matsuno@austintexas.gov will be reviewing things in AMANDA for Mobility Bond Review Discipline
a.	Needs new account training
b.	Reviewer permissions, but not a default for any review. We will re-assign as necessary. 
c.	Ability to Review, Comment and add documents. 
i.	LM
ii.	SP (Read Only, but reviewer permissions at a later date)
iii.	EX
iv.	RW
v.	UC
2.	Sharon.Hamilton@austintexas.gov
a.	Has been a former user of AMANDA
b.	Needs Read only (documents and attachments included)

Thanks!!

**Comment by TracyLinder** [comment]
**Email to Greg:**

Hi Pepper,

I’ve created a DTS ticket for this. I’ll need the following info for both of them please:

•	Employee ID #:
•	Name:
•	Title:
•	Email Address:
•	Phone Number:
•	Supervisor Name:
•	Name of a coworker who has a similar AMANDA account/permissions as you will be needing:
•	Computer log-in name (The name they use to log into their PC)

Thanks,

Tracy Linder, Product Manager

**Comment by TracyLinder** [comment]
**Status:** Nate began working on this a few days ago. Will continue work during sprint 29.

**Comment by Nadin-Nader** [comment]
@TracyLinder The account have been created on TEST and PROD for both users. Waiting on the to log in

**Comment by Nadin-Nader** [comment]
Both accounts have been created. Closing card. If problems arrise. I will create a new card or resolve the issue quickly if possible.
